bin/cli.js

```diff
@@ -52,27 +61,38 @@ if (argv.version) {
   process.exit(0);
 }
 
-argv.headers = argv.H = [].concat(argv.headers).filter(Boolean).reduce(function (headers, header) {
-  header = header.trim().split(':');
-  headers[header[0].trim()] = header[1].trim();
+const opts = {
+  forbidden: argv.forbidden,
 
-  return headers;
-}, {});
+  headers: argv.headers.filter(Boolean).reduce((headers, header) => {
+    header = header.trim().split(':');
+    headers[header[0].trim()] = header[1].trim();
 
-var levels = 'warn info debug'.split(' ');
+    return headers;
+  }, {})
+};
+
+const levels = 'warn info debug'.split(' ');
 bole.output({
   level: levels[argv.verbose] || 'warn',
-  stream: pumpify(ndjs(), process.stdout)
+  stream: pumpify(ndjs(), process.stderr)
 });
-var logger = bole('collapsify-cli');
+const logger = bole('collapsify-cli');
 
-var domain = argv._[0];
+const domain = argv._[0];
 
-require('../')(domain, argv).done(function (output) {
-  console.log('Collapsed Size: ', byte(output.length, {
-    binary: true,
-    digits: 2
-  }));
-}, function (err) {
-  logger.error(err, 'An error has occured while collapsing %s', domain);
-});
+require('..')(domain, opts).then(
+  output => {
+    logger.info(
+      'Collapsed Size: ',
+      byte(output.length, {
+        binary: true,
+        digits: 2
+      })
+    );
+    process.stdout.write(output);
```

Comment:
This was avoided because the output can get rather large. If you must, accept a flag to enable.

Reply:
I usually just redirect output to a file, but yeah, flag sounds good to me.

Reply:
Fixed.